middleware/logger.go

```diff
@@ -24,6 +24,7 @@ func LogrusLogger() gin.HandlerFunc {
 		end := time.Now()
 		duration := end.Sub(start)
 		clientIP := c.ClientIP()
+		user := GetUser(c)
```

Comment:
https://github.com/mikkeloscar/gin-swagger/blob/59a1ef18467addb1701f85beb9c476edf57dfb55/middleware/oauth2.go#L37-L38

Reply:
User data is provided by the `ScopesAuth` middleware
https://github.com/mikkeloscar/gin-swagger/blob/59a1ef18467addb1701f85beb9c476edf57dfb55/middleware/oauth2.go#L8-L10

Reply:
I can make it conditional if its ok to have variable number of fields

Reply:
I think the `uid` field should be somewhat optional because you don't need to have `ScopesAuth` on all routes if you have a public endpoint.

Reply:
Made it optional